spring-integration-mqtt/src/main/java/org/springframework/integration/mqtt/core/Mqttv3ClientManager.java

```diff
@@ -36,13 +39,16 @@
  *
  * @author Artem Vozhdayenko
  * @author Artem Bilan
+ * @author Christian Tzolov
  *
  * @since 6.0
  */
 public class Mqttv3ClientManager
 		extends AbstractMqttClientManager<IMqttAsyncClient, MqttConnectOptions>
 		implements MqttCallbackExtended {
 
+	private final Lock lock = new ReentrantLock();
```

Comment:
There is one already in the super class.
Please, consider to reuse it making `protected`.